packages/patternfly-4/react-integration/README.md

```diff
@@ -1,7 +1,33 @@
-# react-integration
+# Patternfly 4 React Integration Testing
+
+## Overview
+Here you will find a demo application written in typescript, along with cypress integration testing.  This is used to allow developers to verify their code against a working react app that is running their latest PF4 react changes.
+
+New componets should be written in typescript and added to the demo app.  Contributors should write cypress tests to test the functionality of their components.  Contributors should also test all of the user facing interfaces and components that they are creating.
```

Comment:
All new components need to be written in TypeScript and then added to the demo application. 

When making a new PatternFly 4 contribution, you’ll need to write a Cypress test to verify and test functionality. You will also need to test all user facing interfaces and components. 

This README will show you how to launch the demo app and use Cypress to run your tests against it.

Reply:
Done.